Dockerfile

```diff
@@ -16,6 +16,7 @@ ENTRYPOINT ["./docker_entrypoint.sh"]
 HEALTHCHECK --interval=10s --timeout=5s --start-period=1m --retries=3 CMD ./healthcheck.sh
 
 ENV UNLINKED_CAMPAIGNS=1
+ENV PRIORITY_BY_TIME=0
```

Comment:
What order are drops mined in currently?
If it's not a sensible order (after prioritized campaigns are considered), feel free to make this default.

Reply:
They follow the order in the priority list itself

Reply:
They are currently ordered by the priority list's order. 

Setting this to default will change a core behavior of how the bot chooses which drop to mine next.